views.py

```diff
@@ -106,6 +111,11 @@ def download(request, path, rest_call=False, use_async=True, *args, **kwargs):
                         response.content = "<h1>" + content_msg + "</h1>"
                     return response
 
+    elif bag_modified == "true" and metadata_dirty == 'true':
+        if path.endswith("resourcemap.xml") or path.endswith('resourcemetadata.xml'):
+            # we need to regenerate the metadata xml files
+            create_bag_files(res, fed_zone_home_path=res.resource_federation_path)
+
```

Comment:
Yes, I saw that. Just want to make sure this PR is only merged right before the other corresponding HydroShare PR will be merged so that they go together without gap in between.